Criação do CadastraCargo e remoção do CadastrarEditarCargo

NewtonJr5 opened the pull request.

NewtonJr5 committed:
Criação do CadastraCargo e remoção do CadastrarEditarCargo

Itor-Carlos commented:
- Mudanças necessárias:
 * A rota para realizar a alocação no backend está como "alocar_cargo" ao invés de "cargo"
 * No backend as informações extraídas são "idcargo" e "idfuncionario", troque as variaveis do front para serem enviadas com esse nome. Atualmente não está funcionando devido a divergência de nome

NewtonJr5 commented:
É necessário alterar as variaveis de CadastrarCargo?

NewtonJr5 committed:
Mudança em AlocarCargo

Deixei mais parecida com AlocarTarefa, mudei os nomes da variável e mudei a rota no handlesubmit.

NewtonJr5 committed:
Mudança em AlocarCargos

Creio que a rota estava errada.
